### ISSUE TITLE
RAML

### ISSUE THREAD

**Issue by FredericHeem** [lambda2/rapis#2]
What do you think of using [RAML](http://raml.org/) to describe a typical CRUD app according to RAPIS ?

**Comment by lambda2** [comment]
Hello !
I have personally never used RAML, but it looks great!
However, this specification is intended to define how to design a rest API, regardless of the language, format or libraries.

**Comment by FredericHeem** [comment]
Indeed, RAML is great, it generates html api documentation and mock server. The other rest api description languages are _swagger_ and _apiary_ but RAML is my preferred one.
